appcues/src/debug/java/com/appcues/CompositionTestHelper.kt

```diff
@@ -44,7 +44,7 @@ public fun ComposeContent(json: String, imageLoader: ImageLoader) {
     val response = MoshiConfiguration.moshi.adapter(PrimitiveResponse::class.java).fromJson(json)
     val primitive = response!!.mapPrimitive()
 
-    AppcuesTheme {
+    AppcuesExperienceTheme {
```

Comment:
Open to suggestions for naming here, but I wanted to have AppcuesTheme being the theme that respects our branding colors, the Theme we have for experience is just clearing some colors from MaterialTheme so we can use Composable like the Surface and Text.

Reply:
this naming makes sense to me